.gitignore

```diff
@@ -23,3 +23,8 @@ hs_err_pid*
 
 # Maven build directories
 **/target/
+
+#Eclipse Files
```

Comment:
Any chance you can add the defaults for IDEA and Netbeans(!!) too?

https://github.com/kiegroup/kie-wb-common/blob/master/.gitignore#L10

Reply:
@manstis Fixed [here](https://github.com/kiegroup/kie-soup/pull/5).